### ISSUE TITLE
[TextField] Accessibility- Error message not announced in most browser/screen reader combos

### ISSUE THREAD

**Issue by daweffe** [OfficeDev/office-ui-fabric-react#3902]
### Bug Report

- __Package version(s)__: 5.40

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: Planner

#### Describe the issue:

The error message has aria-live="assertive" set, but this does not result in an announcement in all browsers and screen readers.

#### Expected behavior:

Screen readers should announce the alert. I have found adding both role="alert" and aria-live="assertive" works fairly well.

**Comment by dzearing** [comment]
@cschlechty thoughts?

**Comment by cschlechty** [comment]
The occurs because the live region is added simultaneously with the text. Some browsers don't announce text when initially added, and track subsequent changes. One fix would be to always render a container with aria-live, and conditionally render the error. Alerts are live regions by definition, so we shouldn't need aria-live if we go that route.

**Comment by daweffe** [comment]
Just make sure to test with as many screen readers + browser combinations as you can.

**Comment by Jahnp** [comment]
@daweffe , just to understand--is there something that needs to be changed in Fabric React? We don't provide strings by default, but if you're missing a hook/prop to enable this behavior, we're happy to investigate or accept a PR with a fix.

**Comment by daweffe** [comment]
No, I would add it to the div with aria-live=’assertive’

**Comment by aditima** [comment]
@KevinTCoughlin - maybe something you can address while you are working on the TextField?

**Comment by KevinTCoughlin** [comment]
@natalieethell this was the issue that I thought _might_ be solved by the Announcement work? What do you think?
